pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md

```diff
@@ -17,13 +17,13 @@ beta: false
 
 #### Alerting
 
-Custom `alertmanager` [configurations](https://docs.d2iq.com/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) must be copied or re-created [manually](https://docs.d2iq.com/dkp/kommander/2.0/monitoring/#notify-prometheus-alerts-in-slack). You can do this before adapting Prometheus, so that there is no time interval when the alertmanager is not configured.
+Custom `alertmanager` [configurations](/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) must be copied or re-created [manually](/dkp/kommander/2.1/monitoring/#notify-prometheus-alerts-in-slack). You can do this before adapting Prometheus, so that there is no time interval when the `alertmanager` is not configured.
 
 #### Custom ServiceMonitor objects
 
-You should copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons`. This can be done before processing, using different names or namespaces for the new objects. Alternatively, you could just re-label any existing ServiceMonitors before or after the processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are relabeled.
+You should copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons`. This can be done before processing, using different names or namespaces for the new objects. Alternatively, you could just re-label any existing ServiceMonitors before or after the processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.
 
-There are two ways to adapt ServiceMonitors created using Helm values of the Addon:
+You can use two ways to adapt ServiceMonitors created using Helm values of the Addon:
```

Comment:
Let's put this back please. There are is better than 'you can use'.

Reply:
@cmastr - the linter stops us from starting a sentence with "there are"

Reply:
okay - just changed the there are part to the middle, basically: 
ServiceMonitors created using Helm values of the Addon can be adapted in two ways:

Reply:
dang it!! stupid linter. I like your revision.

Reply:
:)